dev/ledger.md

```diff
@@ -368,8 +368,9 @@ A key registration transaction additionally has the following fields:
 The cryptographic hash of the fields above is called the _transaction
 identifier_.  This is written as $\Hash(\Tx)$.
 
-A valid transaction can either be a _signed_ transaction or a _multi-signed_
-transaction.  This is determined by the _signature_ of a transaction:
+A valid transaction can either be a _signed_ transaction, a _multi-signed_
+transaction, or a _logic-signed_ transaction.
```

Comment:
It's probably worth defining logic signatures here, with this spec merge. Something like, 

```
- A logic signature is a [TEAL program](link) along with optional input arguments which returns true upon executing in the execution environment defined by the transaction, the arguments, the round, ... (insert the rest of the things that make up the execution environment). A logic signature is valid if either:
 - The logic signature is signed/multisigned by the transactions' sender, or
 - \Hash(logic signature) = \Sender.
```

Reply:
Probably worth waiting for the multisig spec PR (#6) to get merged first to avoid merge conflicts